src/domain/targeted-messaging/entities/create-outreach.dto.entity.ts

```diff
@@ -1,9 +1,19 @@
+import { DB_MAX_SAFE_INTEGER } from '@/domain/common/constants';
 import { z } from 'zod';
 
 export const CreateOutreachDtoSchema = z.object({
   name: z.string(),
   startDate: z.coerce.date(),
   endDate: z.coerce.date(),
+  sourceId: z
+    .number()
+    .int()
+    .gte(1)
+    .lte(DB_MAX_SAFE_INTEGER - 1),
+  type: z.string(),
+  teamName: z.string(),
+  sourceFile: z.string().nullish().default(null),
+  sourceFileProcessedDate: z.coerce.date().nullish().default(null),
 });
 
 export class CreateOutreachDto
```

Comment:
```suggestion
export class CreateOutreachDto implements z.infer<CreateOutreachDtoSchema>
```

Reply:
It would be `implements z.infer<typeof CreateOutreachDtoSchema>`, right? That's on the next line (20) (https://github.com/safe-global/safe-client-gateway/blob/eb7c106d7fd453178e976bf756c54c4580cb51e5/src/domain/targeted-messaging/entities/create-outreach.dto.entity.ts#L20)

Maybe I'm not understanding your comment, sorry.

Reply:
You're right. My preview window didn't show the next line 🥺